broadcast/internal/arc/arc_submit_tx.go

```diff
@@ -27,6 +33,15 @@ func (a *ArcClient) SubmitTransaction(ctx context.Context, tx *broadcast.Transac
 		opt(options)
 	}
 
+	if options.TransactionFormat == broadcast.RawTxFormat {
+		if err := convertToEfTransaction(tx); err != nil {
```

Comment:
I would move this to `createSubmit*Body()`, but that's just a suggestion

Reply:
Okay, I think I can see why, done.